packages/shared/components/popups/RegisterProposalPopup.svelte

```diff
@@ -30,12 +32,31 @@
             })
             closePopup()
         } catch (err) {
-            if (!nodeUrlError && !eventIdError) {
+            const isAuthenticationError = err?.error?.match(/(username)|(password)|(jwt)/g).length > 0
+            if (isAuthenticationError) {
+                openAuthenticationRequiredPopup()
+            } else if (!nodeUrlError && !eventIdError) {
                 handleError(err)
             }
         }
     }
 
+    function openAuthenticationRequiredPopup(): void {
+        async function onSubmit(auth: Auth): Promise<void> {
+            await registerParticipationEvent(eventId, [{ url: nodeUrl, auth }])
+            showAppNotification({
+                type: 'success',
+                message: localize('views.governance.proposals.successRegister'),
+                alert: true,
+            })
+            closePopup()
```

Comment:
Extract this to a separate function with auth as an optional prop, so that both `handleConfirm` & `openAuthenticationRequiredPopup` can use it.